koku/api/report/queries.py

```diff
@@ -529,11 +529,11 @@ def execute_query(self):
                 units_value = query.values(self.units_key).first().get(self.units_key)
                 if self._count:
                     query_sum = query.aggregate(
-                        value=Sum(self.aggregate_key),
+                        total=Sum(self.aggregate_key),
                         count=Count(self._count, distinct=True)
                     )
                 else:
-                    query_sum = query.aggregate(value=Sum(self.aggregate_key))
+                    query_sum = query.aggregate(total=Sum(self.aggregate_key))
```

Comment:
This will create a response of:
```
{
'total': { 'total': <number>,
              'units': <unit>
          }
}
```
instead of the previous:
```
{
'total': { 'value': <number>,
              'units': <unit>
          }
}
```


Additionally, I believe it will break the UI which was looking for value.

Reply:
Ah, yeah. That would be bad. This made it possible to not special case the "total" block, but I will switch back and re-write some stuff.

Reply:
I'm good if we want to change it, would just need to co-ordinate with UI team, so different PR.